Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Toy/cartridges.yml

```diff
@@ -8,17 +8,17 @@
     caliber: Cap
   - type: Sprite
     netsync: false
-    noRot: true
-    sprite: Objects/Consumable/Trash/ash.rsi
+    directional: true
+    sprite: Objects\Weapons\Guns\Ammunition\Casings\ammo_casing.rsi
     layers:
     - state: base
       map: ["enum.AmmoVisualLayers.Base"]
     drawdepth: FloorObjects
 
 - type: entity
   id: CartridgeCap
-  name: cartridge (cap)
+  name: cap gun cartridge
```

Comment:
Keep the old name (just search "cartridge" in the entity spawn panel to understand why).

Reply:
50/50 on this as I understand cartridge xx is the naming scheme but I like the differentiation that this is specifically a cap gun item.